lib/services/types.proto

```diff
@@ -139,6 +139,9 @@ message ServerV2 {
 
 // ServerSpecV2 is a specification for V2 Server
 message ServerSpecV2 {
+    reserved 8;
```

Comment:
How was this field being used? If it's removed, won't any code relying on it break?

Reply:
It was sourced from `proxy_config.kubernetes.cluster_name` (https://github.com/gravitational/teleport/pull/4354) before we decided to make a dedicated `kubernetes_service`.
Nothing is using it in any released version AFAIK, but I decided to reserve the ID out of abundance of caution.